Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/v1/defaults.go

```diff
@@ -276,17 +276,9 @@ func defaultHostNetworkPorts(containers *[]Container) {
 // default to MustRunAs with namespace allocation.
```

Comment:
update the godoc to clarify that the defaults for missing fields has to be the setting that preserves the behavior that existed prior to the field existing